owa/app/components/admissionLocation/admissionLocationWrapper.js

```diff
@@ -83,8 +78,27 @@ export default class AdmissionLocationWrapper extends React.Component {
                     {}
                 );
 
+                self.setState({admissionLocations: admissionLocations});
+            })
+            .catch(function (errorResponse) {
+                const error = errorResponse.response.data ? errorResponse.response.data.error : errorResponse;
+                self.admissionLocationFunctions.notify('error', error.message.replace(/\[|\]/g, ''));
+            });
+    }
+
+    fetchAllVisitLocations(self) {
+        axios
+            .get(this.urlHelper.apiBaseUrl() + '/location', {
+                params: {
+                    tags: 'Visit Location',
+                    v: 'full'
+                }
+            })
+            .then(function (visitResponse) {
```

Comment:
Same as above. "visitResponse" seems like a misleading name. Would naming something in lines of "visitLocationsResponse" be better?